Solved Question-2 (Merge Sorted Array) of Day-2

Anomasingh opened the pull request:
![Screenshot 2023-12-29 160950](https://github.com/SnowScriptWinterOfCode/LeetCode_Q/assets/118592802/18fb89a0-0446-4269-a877-cc539db0e9b1)
![Screenshot 2023-12-29 161011](https://github.com/SnowScriptWinterOfCode/LeetCode_Q/assets/118592802/a57cc0a6-610f-4052-8574-9a0e7d832eb0)
![Screenshot 2023-12-29 161023](https://github.com/SnowScriptWinterOfCode/LeetCode_Q/assets/118592802/60329fa7-21a8-4fd8-8778-e56f35178b85)
Screenshots of Test Cases Passed

bh-g commented on Day-2/q2.md:
```diff
@@ -0,0 +1,74 @@
+This is the problem
+You are given two integer arrays nums1 and nums2, sorted in non-decreasing order, and two integers m and n, representing the number of elements in nums1 and nums2 respectively.
+
+Merge nums1 and nums2 into a single array sorted in non-decreasing order.
+
+The final sorted array should not be returned by the function, but instead be stored inside the array nums1. To accommodate this, nums1 has a length of m + n, where the first m elements denote the elements that should be merged, and the last n elements are set to 0 and should be ignored. nums2 has a length of n.
+
+Example 1:
+Input: nums1 = [1,2,3,0,0,0], m = 3, nums2 = [2,5,6], n = 3
+Output: [1,2,2,3,5,6]
+Explanation: The arrays we are merging are [1,2,3] and [2,5,6].
+The result of the merge is [1,2,2,3,5,6] with the underlined elements coming from nums1.
+
+Example 2:
+Input: nums1 = [1], m = 1, nums2 = [], n = 0
+Output: [1]
+Explanation: The arrays we are merging are [1] and [].
+The result of the merge is [1].
+
+Example 3:
+Input: nums1 = [0], m = 0, nums2 = [1], n = 1
+Output: [1]
+Explanation: The arrays we are merging are [] and [1].
+The result of the merge is [1].
+Note that because m = 0, there are no elements in nums1. The 0 is only there to ensure the merge result can fit in nums1.
+
+Solution:
+
```
Remove this.

bh-g commented on Day-2/q2.md:
```diff
@@ -0,0 +1,74 @@
+
+Solution:
+
+In C++
```
Make it a comment.

bh-g commented on Day-1/q1-binary-search/ques.md:
Rename the file as solution.cpp and add it here- Day-2/q2: Merge Sorted Arrays

Anomasingh commented:
All Conflicts Resolved -
1. Renamed the file and add it to Day-2/q2: Merge Sorted Array.
2. Removed the question part.
3. Commented out the two text lines

bh-g commented on Day-2/q2: Merge Sorted Arrays/solution.cpp:
```diff
@@ -0,0 +1,51 @@
+// Solution :
+
+// In c++
+
+#include<iostream>
+#include<vector>
+using namespace std;
+class Solution {
+public:
```
Remove this part.

bh-g commented on Day-2/q2: Merge Sorted Arrays/solution.cpp:
```diff
@@ -0,0 +1,51 @@
+        for(i=0; i<m+n; i++)
+            nums1[i] = nums[i];
+    }
+};
```
Delete this.

bh-g commented on Day-2/q2: Merge Sorted Arrays/solution.cpp:
Also, re-indent after making these changes.

Nitya-Pasrija reviewed (changes requested):
Hey @Anomasingh 
There are new changes to the ques.md files. Kindly make the corrections in your PR accordingly :)
